UPGRADE-6.3.md

```diff
@@ -416,10 +416,6 @@ should be quite straightforward.
 Also some menu items that were located in the top bar were now moved to the sidebar. Finally the backend modules that
 were called in a popup window, like navigation, product search index generation and sanity check, now open inline.
 
-From a technical point no changes are required if the project is a shop system. Projects that are pure CMS systems
-should consider removing menu categories that are nevertheless created during migration. Note that there will be some
```

Comment:
Why are shop categories created in non shop environments? That should not be the case.

Reply:
It is no longer the case which is why those lines are removed :-)

Reply:
stuuuupid me! *fetching some coffee